### ISSUE TITLE
Unit Test Setup - Frontend

### ISSUE THREAD

**Issue by Elementator** [amosproj/amos2023ws04-pipeline-manager#63]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been tested
* Feature has been merged into the main branch
* Important parts have been documented in the technology wiki
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release
